app/root.tsx

```diff
@@ -14,7 +19,8 @@ import globalStyles from "./styles/global.css"
 import TopNav from "~/components/topNav";
 
 export const links: LinksFunction = () => [
-  { rel: "stylesheet", href: globalStyles },
+  // { rel: "stylesheet", href: globalStyles },
```

Comment:
Probably, we can create styles/root.css file and import it here.

Reply:
I should remove the commented line, the line below is the auto-generated tailwindcss related file, which is needed to use tailwind.

Reply:
Yeah, definitely you can delete it. I mean if we have some global CSS settings, we can define them in a separate root.css file.

Reply:
will remove